src/store.ts

```diff
@@ -107,6 +107,17 @@ export default class Store extends AinftBase {
     return this.sendRequest(HttpMethod.DELETE, trailingUrl, query);
   }
 
+  giveItemToUser(appId: string, userId: string, itemName: string, quantity: number, reason?: string) {
```

Comment:
Could you add a comment about this function as JsDoc?

Reply:
Added, thanks!